### ISSUE TITLE
VT - Set Up Daily Facilities List Update

### ISSUE THREAD

**Issue by mlighdc** [CDCgov/prime-reportstream#2565]
## Problem statement

Vermont will be using SimpleReport to support their school testing program. They would like to be informed of new facilities  that have signed up for SimpleReport so they can add the facility to their system. Additionally, they would like to add what is typically in the MSH-4 field in the Latest Facilities Report.

## What you need to know

Vermont would like the Latest Facilities Report to be sent daily instead of weekly.

## Acceptance criteria

- [x] Automate the sending of the Latest Facility Report, including the MSH-4 field if possible, on a daily basis to Vermont

## To do

- [ ] Investigate how best to incorporate MSH-4 information (truncated facility name) in the Latest Facility List for Vermont
- [x] Automate the sending of the Latest Facilities Report to Vermont on a daily basis

**Comment by mlighdc** [comment]
@Adrian-Brewster when do you think Vermont can start expecting the daily reports? If you don't think you can add the MSH-4 field before you leave for vacation, let's do that after you get back. 

Would be great if you can start on Thursday, 10/14 before you head out for vacation. Thanks.

**Comment by Adrian-Brewster** [comment]
@mlighdc The report is created and ready to be uploaded to a dashboard and automated.  I can do that before leaving today but I saw your note that this has been completed.  Please confirm so I do not duplicate the report.

**Comment by mlighdc** [comment]
@Adrian-Brewster I did not do anything with the Vermont daily report. Not sure where you saw that I did it. No duplication.

Please send out as soon as possible. Thanks!

**Comment by mlighdc** [comment]
@Adrian-Brewster can you please look at the set up for the Vermont daily report? For some reason, it attached the same file twice. Thanks.

**Comment by Adrian-Brewster** [comment]
@mlighdc  I am not sure how to include MSH-4 on these reports.  It is not an option in Metabase that I am seeing.  I can include facility name and facility CLIA but not the truncated facility name that appears in MSH-4.

**Comment by mlighdc** [comment]
@Adrian-Brewster are you able to show me or tell me what is in Vermont's MSH-4? We can do custom actions (e.g truncation like we can in Excel) in Metabase so we might be able to trunc it.

**Comment by mlighdc** [comment]
@anshulkumar-usds unfortunately, we can't close this one out. We have to address MSH-4 issue so I am re-opening it unless you would like to create a different ticket. I think we can do this via metabase.

**Comment by anshulkumar-usds** [comment]
@Adrian-Brewster looks like the MSH-4 issues is not resolved. Can you please work with VT on this?

**Comment by mlighdc** [comment]
@anshulkumar-usds I already created a test version of the report to include MSH-4. Just sent it off to VT for them to review and see if they have any feedback. If the new report works for them. @Adrian-Brewster just need to update their existing report to include the custom column, which should be pretty easy.

**Comment by Adrian-Brewster** [comment]
@mlighdc Was this addressed when you recreated the dashboard? Can this be closed out now?

**Comment by mlighdc** [comment]
Resolution: I recreated the report and dashboard to automate the sending of daily facilities update to VT. The report includes the MSH-4 field and conversion to SQL helped to stabilize the report and remove special characters. The attached file sent daily is in csv and some of the zip codes are missing the 0 as the first digit of the zip code. This is not a SR issue, this is a csv file format issue. Confirmed with VT and they are comfortable with the format as is but can easily convert to xlsx, if needed in the future for the full zip code if users did not select the proposed address with 9 digit zip code.
